tortoise.py

```diff
@@ -4,7 +4,13 @@ def main():
     FILENAME = 'tiny.tortoise'
     with open(FILENAME, 'r') as tiny_tortoise:
         program = tiny_tortoise.readlines()
-    program = [i.strip() for i in program]
+    # Create a list of lists where each line of commands
+    # from the program are in a list. Strip the new lines
+    # to exclude them as elements in these lists.
```

Comment:
It's also worth explaining why you're taking out the second element of the list.